src/main/java/net/sf/mzmine/modules/peaklistmethods/dataanalysis/mummichog/MummichogTask.java

```diff
@@ -96,9 +97,11 @@ public void run() {
             //arguments[9] = this.mode;
             arguments[8] = "--output";
             arguments[9] = this.output.getAbsolutePath();
+			MummichogParams mummiParameters= new MummichogParams(Double.parseDouble(this.cutoff),this.network,Boolean.parseBoolean(this.force_primary_ion),this.modeling,this.output.getAbsolutePath());
```

Comment:
We probably should change the type of `cutoff` from String to double, `force_primary_ion` from String to boolean, etc.